From e10b4a7e3249100868eeb431320acb3105845fe0 Mon Sep 17 00:00:00 2001
From: Gilles Gouaillardet <gilles@rist.or.jp>
Date: Wed, 22 May 2024 14:39:12 +0900
Subject: [PATCH 2/9] tune_pme: fix MPI/GPU detection

add missing whitespace in order to match the formatting changes
previously done in gmx_print_version_info() with
commit gromacs/gromacs@ffe565fe7a8aef750edf86f08d7f4fe9857e90a1
---
 src/gromacs/tools/tune_pme.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gromacs/tools/tune_pme.cpp b/src/gromacs/tools/tune_pme.cpp
index 965c8b1693..e9f8f49d02 100644
--- a/src/gromacs/tools/tune_pme.cpp
+++ b/src/gromacs/tools/tune_pme.cpp
@@ -690,9 +690,9 @@ static void check_mdrun_works(gmx_bool    bThreads,
 
     /* This string should always be identical to the one in copyrite.c,
      * gmx_print_version_info() in the GMX_MPI section */
-    const char match_mpi[]     = "MPI library:        MPI";
+    const char match_mpi[]     = "MPI library:         MPI";
     const char match_mdrun[]   = "Executable: ";
-    const char match_nogpu[]   = "GPU support:        disabled";
+    const char match_nogpu[]   = "GPU support:         disabled";
     gmx_bool   bMdrun          = FALSE;
     gmx_bool   bMPI            = FALSE;
     gmx_bool   bHaveGpuSupport = TRUE;
-- 
2.39.3

